static/app/views/alerts/rules/metric/types.tsx

```diff
@@ -110,6 +118,8 @@ export type UnsavedMetricRule = {
   monitorWindow?: number | null;
   owner?: string | null;
   queryType?: MEPAlertsQueryType | null;
+  seasonality?: string | null;
+  sensitivity?: string | null;
```

Comment:
rather than using a `string` type here, could we use `AlertRuleSensitivity`? that way we can enforce it's only one of those strings, and it needs to use the enum. that helps a lot if we need to add new levels of sensitivity.